Events-Team.md

```diff
@@ -10,4 +10,25 @@ If you are interested in helping out with events in Oppia and want to join the t
 
 ## Gaming Event server
 
-Oppia has a gaming event server on discord where we usually join for fun nights playing Among Us, Chess, Skribl, etc. Here is a link to join the server: [https://discord.com/invite/dCJwEAgk3D](https://discord.com/invite/dCJwEAgk3D).
\ No newline at end of file
+Oppia has a gaming event server on discord where we usually join for fun nights playing Among Us, Chess, Skribl, etc. Here is a link to join the server: [https://discord.com/invite/dCJwEAgk3D](https://discord.com/invite/dCJwEAgk3D).
+
+### Recording events
+
+OBS is a reliable streaming software that can be used for streaming and recording, which is what was used for the past events at Oppia. Here are the steps to set up OBS:
+
+1.  Install the latest OBS software from [here](https://obsproject.com/).
+
+2.  Open the OBS software and add the following sources by clicking on the "+" icon in the "sources" section in the bottom left part of the screen.
+
+    ![Screenshot showing where sources can be added](images/OBSAddSources.png)
+  - Audio output capture, to capture speaker audio.
+  - Audio input capture, to capture mic audio of organizer.
+  - Window capture of the presented slides.
```

Comment:
```suggestion
    - Audio output capture, to capture speaker audio.
    - Audio input capture, to capture mic audio of organizer.
    - Window capture of the presented slides.
```

Reply:
Do you think placing the bullet points will give the sentence a bit of continuity?

Reply:
Not sure I understood this comment. Did you mean getting rid of the bullet points altogether?

Reply:
Ah, sorry. Do you think placing the bullet points before the image will give the sentence a bit of continuity?

Reply:
Ah, I see. Done!